examples/fit_with_bounds.py

```diff
@@ -43,15 +54,16 @@ def gen_from_expr(self,data, guesses={}):
         fn: function
```

Comment:
the way we have it in master we only return pars?

Reply:
Are you commenting on the docstring here?  If so, it should just describe what the code does.

Reply:
you are supposed to [edit the docstrings to describe the code](https://www.wrike.com/open.htm?id=748442764)